core/src/main/scala/org/apache/spark/sql/execution/columnar/impl/StoreCallbacksImpl.scala

```diff
@@ -184,6 +184,10 @@ object StoreCallbacksImpl extends StoreCallbacks with Logging with Serializable
     list
   }
 
+  override def isInternalBatchTable(tableName: String): Boolean =
```

Comment:
Can you use the existing method isColumnTable() for the same? No need to write this one.